src/screens/CreatePost/index.js

```diff
@@ -106,6 +106,7 @@ const CreatePost = () => {
   const [privacySelect, setPrivacySelect] = React.useState(0);
   const [dataImage, setDataImage] = React.useState([]);
   const [loading, setLoading] = React.useState(false);
+  const [loadingPost, setLoadingPost] = React.useState(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The introduction of `loadingPost` state is a good approach to handle the loading state specifically for post actions. This will help in disabling the button during the post action, improving user experience by preventing multiple clicks.
```diff
+ const [loadingPost, setLoadingPost] = React.useState(false);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->